Week-2/Animation/styles.css

```diff
@@ -0,0 +1,78 @@
+div#main {
```

Comment:
Please prettify the code for easy readability :)

Reply:
yes installed prettify .